docs/cli/tkn-results_pipelinerun_describe.md

```diff
@@ -0,0 +1,54 @@
+## tkn-results pipelinerun describe
+
+Describe a PipelineRun
+
+### Synopsis
+
+Describe a PipelineRun by name or UID. If --uid is provided, then PipelineRun name is optional.
+
+```
+tkn-results pipelinerun describe [pipelinerun-name]
+```
+
+### Examples
+
+```
+Describe a PipelineRun in namespace 'foo':
+    tkn-results pipelinerun describe my-pipelinerun -n foo
+
+Describe a PipelineRun in 'default' namespace:
```

Comment:
In the previous PR, that case was described as `in the current namespace`. Lets try to be consistent. I'm OK with either. Maybe I like the `in the current namespace` a bit more as it is more accurate in the case the user switched from the `default` namespace.

Reply:
I think I missed changing this. Will do the changes.